examples/Training/python/Create_Image.py

```diff
@@ -52,7 +56,7 @@ def planeGen():
 print 'Created new Image:%s Name:"%s"' % (i.getId(), i.getName())
 
 
-# Set the pixel size using units (new in 5.1.0)
+# Set the pixel size using units (new in 5.1.0):
 # =================================================================
 # Lengths are specified by value and a unit enumeration
 # Here we set the pixel size X and Y to be 9.8 Angstroms
```

Comment:
These two lines now render as a continuous sentence so a period after enumeration would be good.